cove_bods/templates/cove_bods/additional_checks_table.html

```diff
@@ -49,7 +49,7 @@
         {% elif additional_check.type == 'entity_statement_not_used_in_ownership_or_control_statement' %}
             <tr>
                 <td>
-                    {% trans 'This Entity Statement is not used in any ownership or control statements.' %}
+                    {% blocktrans %}<code>Entity statement</code> is orphaned. Check whether it should be the <code>subject</code> or <code>interestedParty</code> of a relationship.{% endblocktrans %}
```

Comment:
```suggestion
                    {% blocktrans %}<code>Entity statement</code> is not referenced in any Relationship statements. Check whether it should be the <code>subject</code> or <code>interestedParty</code> of a relationship.{% endblocktrans %}
```
I think orphaned is a bit ambiguous.

Reply:
Yup - will change